src/WPF/ArcGISRuntime.WPF.Samples/Samples/MapView/FeatureLayerTimeOffset/FeatureLayerTimeOffset.xaml.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using System;
+
+namespace ArcGISRuntime.WPF.Samples.FeatureLayerTimeOffset
+{
+    public partial class FeatureLayerTimeOffset
+    {
+        // Hold the feature layer URI
+        private Uri _featureLayerUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0");
+
+        // Hold a reference to the original time extent
+        private TimeExtent originalExtent;
+
+        public FeatureLayerTimeOffset()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create new Map
+            Map myMap = new Map(Basemap.CreateOceans());
+
+            // Create the hurricanes feature layer once
+            FeatureLayer noOffsetLayer = new FeatureLayer(_featureLayerUri);
+
+            // Apply a blue dot renderer to distinguish hurricanes without offsets
+            noOffsetLayer.Renderer = new SimpleRenderer(new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, System.Windows.Media.Colors.Blue, 10));
```

Comment:
The code would read cleaner, if this line were broken into 2 lines of code. One line of code for the creation of the SimpleMarkerSymbol and another to create the SimpleRenderer. Chaining of code together makes it harder to understand what is occurring and requires a lot of scrolling to read what is going on.